web/src/routes/(user)/people/+page.svelte

```diff
@@ -433,6 +436,7 @@
       {#each showPeople as person, index (person.id)}
         <PeopleCard
           {person}
+          {now}
```

Comment:
I tried at first, but if you search for a person and then go back to the full list, you won't use the cached thumbnails

Reply:
This basically is equivalent to not using a cache to begin with, which is not ideal. We just need to reload the one image if/when it changes on the parent page. If you make a fetch request for it in the other component, will that refresh the cache so it is updated by the time you navigate here?

Reply:
`now` in:

| `+page.svelte` | ` people-card` |
| :---: | :---: |
| <video src="https://github.com/immich-app/immich/assets/74269598/d42c47d1-c939-4c4b-86ee-311151da9017">   | <video src="https://github.com/immich-app/immich/assets/74269598/a3351ba7-023a-4637-99aa-df0c93c8765f"> |

It's hard to see in a video, but when thumbnails are not cached it's not smooth

Reply:
> This basically is equivalent to not using a cache to begin with, which is not ideal. We just need to reload the one image if/when it changes on the parent page. If you make a fetch request for it in the other component, will that refresh the cache so it is updated by the time you navigate here?

let me try

Reply:
From my testing, the only way to force the new feature photo to update is to change its thumbnail path (because the image source hasn't changed). So the solution requires a lot more changes (probably adding a new query parameter, parsing its value, updating thumbnailPath...?)

Reply:
Would adding an `updatedAt` parameter fix that issue? If the person has changed, it will have a different `updatedAt` value and the cache will be invalidated. But if they haven't changed, it will be the same value and the thumbnail can stay cached.

Reply:
Yep that fixes it